### PR TITLE
Use virtualenv to build docs

### PR DESCRIPTION
**Pull request by raniejade**
This will allow us to use the latest version of mkdocs and mkdocs-material.

### CODE REVIEW

File: preview_docs.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+BASEDIR=$(dirname "$0")
```
Comment (artem-zinnatullin) [inline preview_docs.sh]: please add `set -e` so that script fails if some command fails, otherwise it'll continue executing

**Review by artem-zinnatullin — APPROVED** [review]

File: build_docs.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+BASEDIR=$(dirname "$0")
```
Comment (vanniktech) [inline build_docs.sh]: Same here

**Review by vanniktech — APPROVED** [review]

**Review by vanniktech — APPROVED** [review]
Out of curiosity why always `.` as a commit message? :dancer:

**Comment by raniejade** [comment]
@vanniktech Less keys to press than `fix something`, the whole PR gets squashed into one commit anyway.

**Comment by vanniktech** [comment]
Yeah thought so.

**Pull request merged**